blueprints/addon-import/index.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
     let blueprintName = options.originBlueprintName;
     let modulePathSegments = [addonName, inflector.pluralize(options.originBlueprintName), fileName];
 
-    if (/-addon/.test(blueprintName)) {
+    if ((/-addon/).test(blueprintName)) {
```

Comment:
We have a lint rule for this?

Reply:
It might be one of the default rules https://eslint.org/docs/rules/wrap-regex.

Reply:
I'm confused. If it's not already failing on master, then it must not be in our ruleset.